src/content/documentation/publish/third-party-library-usage.md

```diff
@@ -15,11 +15,14 @@ date: 2022-10-17
 
 # Third Party Library Usage
 
-To complete the review process at [addons.mozilla.org](https://addons.mozilla.org) (AMO), reviewers must be able to verify the code in your extension. You may include third-party libraries in your extension. In that case, when you upload your extension to AMO, you will need to provide links to the library source code.
+The [Development Practices](/documentation/publish/add-on-policies/#development-practices) section of the [Add-ons Policies](/documentation/publish/add-on-policies/) states that you can include third party libraries in your extension as long the following conditions are met:
 
-If your add-on uses minified, obfuscated or otherwise machine-generated first-party code, please see [our requirements](/documentation/publish/source-code-submission/) for that.
+* Extensions may only use the release versions of third party libraries.
+* Third party libraries may not be modified.
```

Comment:
This is still duplicating content from the policies page. We should avoid that because we can't re-check all related pages when the policies are updated to find copies of provisions that were updated. Wouldn't a link to the policies section be enough, as you do in line 18?

Reply:
No, I don't think so. This PR was created in response to a request from @mixedpuppy to quote the policy text on this page. The initial motivation for that request was that this page did not communicate the policy requirements to the reader despite the positioning that it is authoritative on the topic.

The other approach I was considering was to move the relevant policy text out of `addons-policy.md` and into a [partial](https://liquidjs.com/tutorials/options.html#Partials-Layouts). That partial would then be imported in both `addons-policy.md` and `third-party-library-usage.md`. But since we don't have an existing pattern of using partials like this I held off.

Reply:
As I said in private, this is a best practices document, not a policies or requirements document. We should not duplicate official requirements into best practices documents. We can link to the policies where the requirements are stated, but the duplication is a recipe for failure, and we have to avoid that for content that touches policy provisions. There needs to be a single source of truth that we can we can reasonably keep up to date.